styles/tree-view.less

```diff
@@ -45,10 +45,20 @@
   padding-right: @component-padding;
   position: relative;
 
-  // This fixes #110, see that issue for more details
-  .entry:before {
-    content: '';
-    position: absolute;
+  .entry {
+    .name {
+      pointer-events: none;
+    }
+
+    &.dragenter {
+      background-color: rgba(255, 255, 255, 0.1);
```

Comment:
The .dragenter class is not used anymore, since I'm just adding the `selected` class to the DirectoryView - I'll remove that